### ISSUE TITLE
Tests may be using low precision

### ISSUE THREAD

**Issue by taldcroft** [sot/Ska.quatutil#5]
E.g. this test:
```
def test_radec2yagzag():
    yag, zag = Ska.quatutil.radec2yagzag(ra2, dec2, q0)
    assert_almost_equal(yag, 0.35751029916939936)
    assert_almost_equal(zag, -0.32107186086370215)
```
This is using nosetools `assert_almost_equal`.  The docs page for this doesn't seem to load, but my guess is that this will be comparing to abs=1e-6 and rel=1e-6, which in degrees isn't much.

This came from https://github.com/sot/proseco/issues/36 where there was a test fail on Windows that is fundamentally an issue with `Ska.quatutil`.

**Comment by jeanconn** [comment]
Is the action to improve the precision or to get rid of nose ?

**Comment by taldcroft** [comment]
Yes.  :smile:  And then maybe we will see test failures on Windows that can be debugged.
